wazo_calld/plugins/meetings/bus_consume.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2019-2021 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+
+from wazo_calld.plugin_helpers.confd import Meeting
+from wazo_calld.plugin_helpers.exceptions import NoSuchMeeting
+
+from .schemas import participant_schema
+
+logger = logging.getLogger(__name__)
+
+
+class MeetingsBusEventHandler:
+    def __init__(self, confd, notifier, service):
+        self._confd = confd
+        self._notifier = notifier
+        self._service = service
+
+    def subscribe(self, bus_consumer):
+        bus_consumer.on_ami_event('ConfbridgeJoin', self._notify_participant_joined)
+        bus_consumer.on_ami_event('ConfbridgeLeave', self._notify_participant_left)
+
+    def _notify_participant_joined(self, event):
+        meeting_uuid = self._extract_meeting_uuid(event['Conference'])
```

Comment:
This will get called for all confbridge events. We should check if it starts with `wazo-meeting-` before doing anything to avoid getting gibberish from `_extract_meeting_uuid` or raising an error